lxd/request/const.go

```diff
@@ -45,6 +45,9 @@ const (
 
 	// CtxTrusted is a boolean value that indicates whether the request was authenticated or not.
 	CtxTrusted CtxKey = "trusted"
+
+	// CallbackFunc is a callback function that can be called on the end of a request handler on an operation.
+	CallbackFunc CtxKey = "callback_function"
```

Comment:
This change shouldnt be part of this commit as its not related to the operation onDone callback functionality